profiling/src/crashtracker/crash_info.rs

```diff
@@ -30,20 +30,20 @@ pub struct StackFrame {
 
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct SigInfo {
-    signum: u64,
-    signame: Option<String>,
+    pub signum: u64,
```

Comment:
why do these need to be `pub`

Reply:
I'm reading them from the profiling-crashtracking-receiver crate to create the telemetry message